app/src/main/java/edu/uc/haflga/financialpal/dto/Category.kt

```diff
@@ -9,7 +9,7 @@ class Category(
     var id: Int = 0
 
     //Creating an ArrayList for transaction Categories
-    var allCategories = ArrayList<Category>()
+    var allCategories = mutableListOf<Category>()
```

Comment:
ArrayList and mutableListOf should both end up with a final container of ArrayList<Category> but mutableListOf should provide much for functionality and potential to the container.

Reply:
Hmmm.  I had to look this one up: https://stackoverflow.com/questions/43114367/difference-between-arrayliststring-and-mutablelistofstring-in-kotlin 
Interesting.